README.md

```diff
@@ -126,7 +128,7 @@ To ensure you have all the correct packages installed locally in your dev enviro
 make install
 ```
 
-From your bash compatible shell.  This will install all of the necessary vendored libraries that the project uses to
```

Comment:
Can you please reduce this to:
"This will install the library dependencies to`the /vendor` directory.